include/libdnf/rpm/package.hpp

```diff
@@ -47,11 +47,22 @@ class Transaction;
 
 namespace libdnf::rpm {
 
+struct PackageId {
```

Comment:
I am not sure whether it is the best place where to put PackageId(). PackageId was and will be used in solv part, where include of package.hpp can be unwanted.

Reply:
Can you point out where exactly is the problem? I didn't actually have to add an include for this in anything in `libdnf/rpm/solv`. there's an `#include "libdnf/rpm/package.hpp"` in `libdnf/rpm/solv/package-private.hpp` which seems to have been there since the beginning.

Unless there's an immediate problem I'd keep the `PachageId` here and if we ever need, it can be moved later?

Reply:
At the present time there is no problem. And in future we can move it again.